work14/index.js

```diff
@@ -0,0 +1,116 @@
+"use strict";
+const wrap = document.getElementById("js_wrap");
+const ul = document.getElementById("js_list");
```

Comment:
IMO
I think you should avoid making variables used in a particular function global variables.
What do you think?

Reply:
I didn't need to make it a global variable because I only use it in specific places.

I saw the code of mai-san, but it doesn't use global variables. I learned a lot.

thank you.


commit: [a5e02ec](https://github.com/yasu-pro/js-handson/pull/16/commits/a5e02ecd965d8176dd7a02f3a09124e881e6efdb)